jobs/config.json

```diff
@@ -11192,6 +11192,51 @@
       "sig-aws"
     ]
   },
+  "pull-kops-verify-bazel": {
+    "args": [
+      "./hack/verify-bazel.sh"
+    ],
+    "scenario": "execute",
+    "sigOwners": [
+      "sig-lifecycle"
+    ]
+  },
+  "pull-kops-verify-boilerplate": {
+    "args": [
+      "./hack/verify-boilerplate.sh"
+    ],
+    "scenario": "execute",
+    "sigOwners": [
+      "sig-lifecycle"
+    ]
+  },
+  "pull-kops-verify-gofmt": {
+    "args": [
+      "./hack/verify-gofmt.sh"
+    ],
+    "scenario": "execute",
+    "sigOwners": [
+      "sig-lifecycle"
+    ]
+  },
+  "pull-kops-verify-govet": {
+    "args": [
+      "make govet"
```

Comment:
the args need to be separated, do "make", "govet"

Reply:
👍